src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,29 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao) injector
+                .getInstance(ManufacturerDao.class);
+        Manufacturer zaz = new Manufacturer("ZAZ", "Ukraine");
+        Manufacturer hyundai = new Manufacturer("Hyundai", "Korea");
+        Manufacturer volvo = new Manufacturer("Volvo", "Sweden");
+        Manufacturer bmw = new Manufacturer("BMW", "Germany");
+        manufacturerDao.create(zaz);
+        manufacturerDao.create(hyundai);
+        manufacturerDao.create(volvo);
+        manufacturerDao.create(bmw);
+        manufacturerDao.getAll()
+                .stream()
+                .forEach(System.out::println);
```

Comment:
This code can be simplified
```suggestion
        manufacturerDao.getAll().forEach(System.out::println);
```
Fix all cases please.

Reply:
I thought it was more readable.